### ISSUE TITLE
PHPTAL  Version Info update?

### ISSUE THREAD

**Issue by miwebguy** [phptal/PHPTAL#92]
<?php
/**
 * PHPTAL templating engine
 *
 * PHP Version 5
 *
 * @category HTML
 * @package  PHPTAL
 * @author   Laurent Bedubourg <lbedubourg@motion-twin.com>
 * @author   Kornel Lesiński <kornel@aardvarkmedia.co.uk>
 * @license  http://www.gnu.org/licenses/lgpl.html GNU Lesser General Public License
 * @version  SVN: $Id$
 * @link     http://phptal.org/
 */

define('PHPTAL_VERSION', '1_3_1');

**Comment by Ocramius** [comment]
I'd probably rather drop the value, or set it to something like `check-your-composer-lock-file` :D 

Alternative, `composer-runtime-api` declares the current versions

**Comment by miwebguy** [comment]
That's probably ok. I have an older implementation that only uses the classes folder.

**Comment by Ocramius** [comment]
Yeah, dropping the constant would be a major BC break: setting it to a default would be better

**Comment by Potherca** [comment]
As an alternative approach, we could add a GitHub Action (GHA) to change the value of the constant when a tag is created?
If auto-commit is not desirable, it would be trivial to make it an MR instead...
